parquet/src/main/java/org/apache/iceberg/parquet/Parquet.java

```diff
@@ -339,6 +344,8 @@ private static class Context {
       private final int rowGroupSize;
       private final int pageSize;
       private final int pageRowLimit;
+
```

Comment:
Nit: don't think we need the newline here?

Reply:
ACK